mission-control-client/src/actions/activeProjectActions.js

```diff
@@ -1,7 +1,24 @@
+import { peopleByProjectId } from "../queries";
+import axiosLabsGraphQL from "../utils/axiosLabsGraphQL";
+
 export const SET_ACTIVE_PROJECT = "SET_ACTIVE_PROJECT";
+export const SET_ACTIVE_PROJECT_SUCCESS = "SET_ACTIVE_PROJECT_SUCCESS";
+export const SET_ACTIVE_PROJECT_FAILURE = "SET_ACTIVE_PROJECT_FAILURE";
 
 export const setActiveProject = el => {
   return dispatch => {
-    dispatch({type: SET_ACTIVE_PROJECT, payload: el})
-  }
-}
+    dispatch({ type: SET_ACTIVE_PROJECT });
```

Comment:
We could do SET_ACTIVE_PROJECT_START when we do a code cleanup to avoid confusion